bungee/src/main/java/codecrafter47/bungeetablistplus/placeholder/PlayerPlaceholderResolver.java

```diff
@@ -108,9 +109,18 @@ public PlayerPlaceholderResolver(ServerPlaceholderResolver serverPlaceholderReso
         addPlaceholder("SimpleClans_ClanColorTag", create(BukkitData.SimpleClans_ClanColorTag));
         addPlaceholder("vault_primary_group", create(MinecraftData.Permissions_PermissionGroup));
         addPlaceholder("vault_prefix", create(MinecraftData.Permissions_Prefix));
+        addPlaceholder("vault_prefix_color", create(MinecraftData.Permissions_Prefix, p -> {
+            return ChatFormat.lastFormatCode(p);
+        }, TypeToken.STRING));
         addPlaceholder("vault_suffix", create(MinecraftData.Permissions_Suffix));
         addPlaceholder("vault_primary_group_prefix", create(MinecraftData.Permissions_PrimaryGroupPrefix));
+        addPlaceholder("vault_primary_group_prefix_color", create(MinecraftData.Permissions_PrimaryGroupPrefix, p -> {
+            return ChatFormat.lastFormatCode(p);
+        }, TypeToken.STRING));
         addPlaceholder("vault_player_prefix", create(MinecraftData.Permissions_PlayerPrefix));
+        addPlaceholder("vault_player_prefix", create(MinecraftData.Permissions_PlayerPrefix, p -> {
```

Comment:
The color suffix is missing in the placeholder name.

Reply:
Fixed, sorry about that